scripts/ofp_linux_interface_release.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Use this script to restore access to linux interface for other
+# applications after was blocked with ofp_linux_interface_acquire.sh.
```

Comment:
Typos: `a linux interface` and missing `it`

Reply:
OK